fetch/api/request/request-cache-force-cache.html

```diff
@@ -71,6 +71,15 @@
         expected_validation_headers: [false],
         expected_no_cache_headers: [false],
       },
+      {
+        name: 'RequestCache "force-cache" mode checks the cache for previously cached content and goes to the network if previous response was no-store',
+        state: "stale",
+        cache_control: "no-store",
+        pragma: "no-cache",
+        request_cache: ["default", "force-cache"],
```

Comment:
The case stale/no-store/default is probably already covered in fetch/api/request/request-cache-default.html.
We probably can remove it.

Reply:
The purpose of this added test case is that `force-cache` does not override the browsers behavior of suppressing no-store responses that may be hanging around in the browser.  For example, the browser may still have this data resident to support devtools viewsource, etc.  Setting `force-cache` should not allow content to observe these `no-store` responses.

Reply:
Sure, instead of request_cache: ["default", "force-cache"], request_cache: ["force-cache"] should be sufficient.

Reply:
Don't I need the "default" there so that the http cache has a chance to see the no-store response?  Or does the test infrastructure prime cache in some other way?  Thanks.